pyproject.toml

```diff
@@ -38,7 +38,10 @@ select = ["E", "W", "F", "C", "N", "R", "D", "H"]
 # Ignore W503, E501 because using black creates errors with this
 # Ignore D107 Missing docstring in __init__
 ignore = ["W503", "E501", "D107"]
-# D100, D101, D102, D103: Ignore missing docstrings in tests
+# D100, D101, D102, D103, D104: Ignore missing docstrings in tests
+# D205: Ignore 1 blank line required between summary line and description in tests
```

Comment:
Can't we just adapt out docstring to comply? Or do we not want to follow these?

Reply:
From the [Contributing](https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#test-structure), this is the docstring to be used in tests
```
    """
    arrange: given 2 numbers
    act: when they are added
    assert: the result must be the sum of the 2 numbers. If a description spans
        multiple lines, lines after the first should be indented by the default
        spacing of the file to indicate that the description continues.
    """
```
Flake8 is not so happy about it.